python/cudf/dataframe/series.py

```diff
@@ -287,6 +287,29 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
         else:
             return NotImplemented
 
+    def __array_function__(self, func, types, args, kwargs):
+
+        cudf_series_module = Series
+        for submodule in func.__module__.split(".")[1:]:
+            # point cudf to the correct submodule
+            if hasattr(cudf_series_module, submodule):
+                cudf_series_module = getattr(cudf_series_module, submodule)
+            else:
+                return NotImplemented
+
```

Comment:
Might want to  add a check for type here like  in cupy: https://github.com/cupy/cupy/blob/master/cupy/core/core.pyx#L1303-L1306 

Was not sure if we only wanted to only limit np functions with cudf_series as arguments.  (for arguments that have __array_function__ like np/cupy/dask arrays etc) . 
```python
handled_types = [cudf_series_module]
for t in types:
   if t not in handled_types:
     return NotImplemented
```

Reply:
Yes we should limit it for now at least.

Reply:
Fixed with 303d31d .
Sorry for the rebase but with the `dask-cudf` I felt it was the cleanest way to resolve conflicts.